scripts/ingest-tarball.sh

```diff
@@ -59,7 +59,7 @@ fi
 
 # Ingest the tarball to the repository
 echo "Ingesting tarball ${tar_file} to ${repo}..."
-${decompress} "${tar_file}" | cvmfs_server ingest -t - -b "${version}" "${repo}"
+${decompress} "${tar_file}" | cvmfs_server ingest -t - -b versions/"${version}" "${repo}"
```

Comment:
Wouldn't just `versions` make more sense, so we can let the top-level directory in the tarball to ingest be `2021.12/` (rather than `compat` or `software`)?

Reply:
```suggestion
${decompress} "${tar_file}" | cvmfs_server ingest -t - -b versions "${repo}"
```

Reply:
Okay with me, but that will require some more work in both this script (as it now assumes that the top-level dir in the tarball is `{software,compat,init}`, and also does some checks based on what it finds in the tarball) and the one that creates the tarballs.

Reply:
This should work now for tarballs that have contents like `2021.12/{compat,software,init}/..../....`. The script checks if the first two levels in the directory structure of the contents match with the corresponding parts in the filename (`eessi-2021.12-compat-.....-<timestamp>.tar.gz`)